tests/integration/workflow/test_test_suite.py

```diff
@@ -262,3 +262,15 @@ def test__load_test_samples(
         (test_case_1, sorted([dummy_test_samples[0], dummy_test_samples[1]])),
         (test_case_2, sorted([dummy_test_samples[2], dummy_test_samples[1]])),
     ]
+
+
+def test__init__validate_name() -> None:
```

Comment:
Nit: we can move these to unit tests with slight modification by not including testcase.

Reply:
Will do for similar tests as well.